library/src/main/java/com/google/maps/android/data/Renderer.java

```diff
@@ -326,9 +363,9 @@ public Feature getContainerFeature(Object mapObject) {
     /**
      * Gets the URLs stored for the Marker icons
      *
-     * @return mMarkerIconUrls ArrayList of URLs
+     * @return mMarkerIconUrls Set of URLs
      */
-    public ArrayList<String> getMarkerIconUrls() {
```

Comment:
Noting that this is another breaking change. I do think this is a break API though.

Reply:
As before, I don't believe anyone would actually be using the `Renderer` API directly. I've made this particular method protected now under this assumption. The previous `ArrayList` was always only adding unique icon URLs, but using a set makes this much more efficient, avoiding full list scanning for `contains()` each time one is added.